moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -302,6 +302,76 @@ bool MoveItConfigData::outputOMPLPlanningYAML(const std::string& file_path)
 
   emitter << YAML::Value << YAML::BeginMap;
 
+  std::vector<OMPLPlannerDescription> planner_des = getOMPLPlanners();
```

Comment:
I did not move this block, but rather I took the list of OMPL planners out and made a separate function to generate the list.  I guess Git thought this was the right diff to show rather than showing an even larger block of OMPL planners being copied and pasted elsewhere.

Reply:
I did not move this block, but rather I took the list of OMPL planners out and made a separate function to generate the list. I guess Git thought this was the right diff to show rather than showing an even larger block of OMPL planners being copied and pasted elsewhere.  

I created the `getOMPLPlanners()` function so that I could get the list of planners to populate the default planner combo box.  I looked everywhere, but this was the only location that contained the OMPL planner list, and the function `outputOMPLPlanningYAML()` did not return the list, but wrote it to a YAML file.